include/electionguard/ballot.hpp

```diff
@@ -19,14 +19,30 @@ namespace electionguard
     /// Enumeration used when marking a ballot as cast or spoiled
     /// </summary>
     enum class BallotBoxState {
+        /// <summary>
+        /// A ballot whose state is unknown to ElectionGuard
+        /// and will not be included in any election results
+        /// </summary>
+        notset = 0,
         /// <summary>
         /// A ballot that has been explicitly cast
         /// </summary>
         cast = 1,
         /// <summary>
-        /// A ballot that has been explicitly spoiled
+        /// A ballot that never included in the tally BUT included in election record
+        /// - contents always decrypted and stored in plain text
+        /// - always included in the verification site
+        /// - used to be called spoiled
         /// </summary>
-        spoiled = 2,
+        challenged = 2,
+        /// <summary>
+        /// A ballot that never included in the tally
+        /// - included in election record for verification
+        /// - contents never decrypted
+        /// - never included in the verification site
+        /// - slightly new meaning of what spoiled is
+        /// </summary>
+        spoiled = 3,
```

Comment:
These are the accidents that will not be in the tally but will be added to the election record so that they can be verified.